pylibfdt/meson.build

```diff
@@ -6,7 +6,7 @@ pylibfdt = custom_target(
   input: 'libfdt.i',
   depends: libfdt,
   output: '_libfdt.so',
-  command: [setup_py, 'build_ext', '--build-lib=' + meson.current_build_dir()],
```

Comment:
So you remove `--build-lib` here, but it's still used in the Makefile build.  Is that intentional?

Reply:
Nope, I forgot to check the Makefiles. It has the same issue with rebuilding between a `make all` -> `make install`. Removing `--build-lib` fixes it there too. Pushed a second commit to fix Makefile.